pkg/services/auth/jwt/jwt.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
Now you lost me a bit. Could this potentially be done in a separate PR to reduce scope of changes? To understand this I would need a PR description or a design doc discussion to understand this.

Reply:
:+1: moved to https://github.com/grafana/grafana/pull/57420

Reply:
Think you can add more context in the https://github.com/grafana/grafana/pull/57420 description?

Reply:
@marefr yep, will do